tests/Makefile

```diff
@@ -355,6 +355,10 @@ test-gzstd: gzstd
 test-zstdgrep: gzstd
 	@echo a | $(PRGDIR)/zstd | $(PRGDIR)/zstdgrep a
 	@echo a | $(PRGDIR)/zstd | $(PRGDIR)/zstdgrep b && return 1 || return 0
+	-[ -f zstdcat ] || ln -s $(PRGDIR)/zstd zstdcat
```

Comment:
The test seems to leave a `zstdcat` link in the working directory.
Is that intentional and beneficial ?
The links seems only useful for the test itself.

Reply:
The `test-lz4` test does leave `lz4` in cwd too. But I'd move `zstdcat` link to `/tmp`.

Reply:
Yep, that's a good choice